source/fundamentals/builders/indexes.txt

```diff
@@ -11,3 +11,150 @@ Indexes Builders
    :class: singlecol
 
 .. _indexes-builders:
+
+Overview 
+--------
+
+This guide shows you how to specify **indexes** using the Java Driver
+:doc:`builders </fundamentals/builders/>`. Index builders provide
+convenient methods for different types of indexes:
+
+- :ref:`Ascending <index-ascending>`
+- :ref:`Descending <index-descending>`
+- :ref:`Compound Indexes <index-compound-indexes>`
+- :ref:`Text Indexes <index-text-indexes>`
+- :ref:`Hashed Indexes <index-hashed-indexes>`
+- :ref:`Geospatial Indexes <geospatial-indexes>`
+
+Like an index in a textbook, an index in MongoDB groups data based
+on the field you provide and by an index builder mentioned above. See
+the :doc:`Indexes </fundamentals/indexes>` page to see examples of
+queries on indexes. 
+
+The :java-core-api:`Indexes <com/mongodb/client/model/Indexes.html>`
+class provides static factory methods for all the MongoDB Index key
+types. Each method returns a :ref:`Bson <bson>` instance, which you can
+pass to `createIndex() <https://mongodb.github.io/mongo-java-driver/4.2/apidocs/mongodb-driver-sync/com/mongodb/client/MongoCollection.html#createIndex(com.mongodb.client.ClientSession,org.bson.conversions.Bson)>`_. 
+
+.. tip::
+
+   For brevity, you may choose to import all methods of the
+   :java-core-api:`Indexes <com/mongodb/client/model/Indexes.html>`
+   class statically: 
+
+   .. code-block:: java
+
+      import static com.mongodb.client.model.Indexes.*;
+
+   The examples below assume this static import.
+
+.. _index-ascending:
+
+Ascending
+---------
+
+Pass the field name you want to index to the 
+:java-core-api:`ascending() <com/mongodb/client/model/Indexes.html#ascending(java.lang.String...)>`
+method to create an ascending index. An ascending index will sort the
```

Comment:
Let's avoid using the future tense (identified here by "will") when possible as the style guide asks to use present tense https://docs.mongodb.com/meta/style-guide/writing/use-present-tense/:

```suggestion
method to create an ascending index. An ascending index sorts the
```

Applies to all instances.